modules/Miss.js

```diff
@@ -40,12 +40,18 @@ class Miss extends React.Component {
     return (
       <LocationSubscriber>
         {(location) => {
-          const { render, component:Component } = this.props
+          const { children, render, component:Component } = this.props
           const { noMatchesInContext } = this.state
           const { serverRouter, match } = this.context
           const noMatchesOnServerContext = serverRouter &&
             serverRouter.missedAtIndex(match.serverRouterIndex)
-          if (noMatchesInContext || noMatchesOnServerContext) {
+          const matched = !(noMatchesInContext || noMatchesOnServerContext)
```

Comment:
The `matched` passed to `Match`'s `children` only indicates if _that particular_ `Match` was matched.  If that wasn't the case it wouldn't be very useful.  `Miss` is supposed to "match" when no `Match`es "matched", so it's inverse.  Basically, if it a particular `Match` or `Miss` should render stuff then it should set `matched` to `true`.